packages/eslint-plugin-template/src/rules/headings-consecutive-order.ts

```diff
@@ -0,0 +1,82 @@
+import {
+  TmplAstElement,
+  TmplAstTemplate,
+} from '@angular-eslint/bundled-angular-compiler';
+import { createESLintRule } from '../utils/create-eslint-rule';
+import { getTemplateParserServices } from '@angular-eslint/utils';
+import type { TmplAstNode } from '@angular/compiler';
+
+export const RULE_NAME = 'headings-consecutive-order';
+export const HEADING_TAGS = ['h1', 'h2', 'h3', 'h4', 'h5', 'h6'];
+export type MessageId = 'headingsOrder';
+
+export default createESLintRule<[], MessageId>({
+  name: RULE_NAME,
+  meta: {
+    type: 'suggestion',
+    docs: {
+      // link: https://www.w3.org/WAI/tutorials/page-structure/headings/
+      description:
+        'Ensures that headings have consecutive order from largest to lowest h1 -> h6',
```

Comment:
We're looking to follow a convention where a11y rules contain the prefix `[Accessibility]` in the description (example [here](https://github.com/angular-eslint/angular-eslint/blob/main/packages/eslint-plugin-template/src/rules/valid-aria.ts#L30C12-L30C12)). Please add that here.